scripts/webpack/webpack.common.ts

```diff
@@ -86,7 +86,8 @@ module.exports = {
         // However, we still need to prefix the public URL with /assets/static/img
         options: {
           outputPath: 'static/img',
-          publicPath: '/assets/static/img',
+          // using relative path to make this work when pyroscope is deployed to a subpath (with BaseURL config option)
+          publicPath: '../assets/static/img',
```

Comment:
This is going to work as long as images are being loaded from the CSS, ie if we add an image fromthe html it will fail to load since it's coming from the root.

But it should be fine for now, since we want to [address this in the future anyway](https://github.com/pyroscope-io/pyroscope/tree/base-url-test)?